agent/agent.go

```diff
@@ -184,35 +192,46 @@ func (a *Agent) RunSingleInput(inputConfig *models.InputConfig, ctx context.Cont
 		a.gatherLoop(ctx, acc, input, ticker, interval)
 	}(input)
 
-	a.Config.Inputs = append(a.Config.Inputs, input)
+	alreadyInArray := false
```

Comment:
Probably make a contains function that checks this / see if they have one in a utils file somewhere or something.

Reply:
they have a sliceContains function that checks if string exists in []string.  Will implement our own.